src/App/Controls/DismissModalToolBarItem.cs

```diff
@@ -15,12 +16,23 @@ public DismissModalToolBarItem(ContentPage page, string text = null, Action canc
             // TODO: init and dispose events from pages
             InitEvents();
             Text = text ?? AppResources.Close;
+            Icon = Helpers.ToolbarImage("ion_chevron_left.png");
```

Comment:
Looks like you don't need anything here anymore since you are not using modals for windows?

Reply:
There are still some modals being used.

For example all the settings page actions are still modals. If they aren't they don't come up above the entire screen and look a bit weird. So there are still some close/cancel buttons